ca/testdata/ca.json

```diff
@@ -6,7 +6,7 @@
     "password": "password",
     "address": "127.0.0.1:0",
     "dnsNames": ["127.0.0.1"],
-    "logger": {"format": "text"},
+    "_logger": {"format": "text"},
```

Comment:
I guess these disable logging for tests? If that's indeed the goal, then it's OK 🙂

Reply:
Yes, it's hard to see what's going on if something fails. You can easily enable it for debugging purposes locally.